Worksheet-c/readme.md

```diff
@@ -0,0 +1,21 @@
+Worksheet C:
+
+Question a :
+Checks if values in a list are duplicated by checking one value in the list against the rest, then the next value and so on. Also it checks if I and j are the same so that the same position in the list isn’t checked as it would return true when it isn’t what is being looked for. 
+Question b:
+Because it checks each number in the list n-1 amount of times before checking each number again for another lot of n-1.
+Question c:
+Each while loop of j only increases i by one. Therefore i is the same as n within the j for loop.
+Question d:
+Half as many variables to deal with?
```

Comment:
Kind of -- actually half as many pairings, as each pairing is checked only once rather than twice.